core/pom.xml

```diff
@@ -214,14 +215,14 @@
 
         <!-- File upload -->
         <dependency>
-            <groupId>commons-fileupload</groupId>
-            <artifactId>commons-fileupload</artifactId>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-fileupload2-jakarta</artifactId>
+            <version>2.0.0-M1</version>
```

Comment:
Can you please use
```
        <dependency>
            <groupId>org.apache.commons</groupId>
            <artifactId>commons-fileupload2-jakarta-servlet6</artifactId>
            <version>2.0.0-M2</version>
        </dependency>
```
instead?

Reply:
Feel free to open a PR :)

Reply:
In progress https://github.com/apache/struts/pull/861

Reply:
Thanks Lukasz, I still had it on my list for this week.